packages/client/modules/demo/ClientGraphQLServer.ts

```diff
@@ -248,7 +248,7 @@ class ClientGraphQLServer extends (EventEmitter as GQLDemoEmitter) {
         }
       }
     },
-    TaskFooterIntegrateMenuRootQuery: (_teamId: unknown, userId: string) => {
+    TaskFooterIntegrateMenuQuery: (_teamId: unknown, userId: string) => {
```

Comment:
These queries were recently renamed. Trying to integrate a task card with an integration or ending a demo causes a crash

Reply:
Just curious, `ops` here refers to simulated operations? The query result here is actually mocked, so it needs to be consistent with the query statement returned by the real GraphQL API. Is my understanding correct? 😁